lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -189,7 +189,12 @@ def store(payment_method, options = {})
 
         # If customer option is provided, create a payment method and attach to customer id
         # Otherwise, create a customer, then attach
-        if payment_method.is_a?(StripePaymentToken) || payment_method.is_a?(ActiveMerchant::Billing::CreditCard)
+        card_source_pay = payment_method.source.to_s if defined?(payment_method.source)
+        card_brand_pay = card_brand(payment_method) unless payment_method.is_a?(String) || payment_method.nil?
+        if NO_WALLET_SUPPORT.include?(card_source_pay) || NO_WALLET_SUPPORT.include?(card_brand_pay)
+          result = super(payment_method, options)
+          return result if result.is_a?(ActiveMerchant::Billing::Response)
```

Comment:
What are the other potential outcomes here other than an AM Response? And what happens if it's not?